tests/BuildScriptGeneratorCli.Tests/BuildCommandTest.cs

```diff
@@ -224,7 +224,8 @@ public void DoesNotShowHelp_EvenIfIntermediateDir_DoesNotExistYet()
         // sure wouldn't change. Since we cannot update product code with test generator we cannot run this test in
         // a docker container. So we run this test on a Linux OS only as build sets execute permission flag and
         // as well as executes a bash script.
-        [EnableOnPlatform("LINUX")]
+        [Fact (Skip = "Skipping test temporarily")]
```

Comment:
why?

Reply:
Accidental addition from merging branches. Revert of this has been made now